### ISSUE TITLE
RESET function not working with new version 1.37.0

### ISSUE THREAD

**Issue by MostafaaRamadan** [umami-software/umami#1416]
with new version 1.37.0 when I try to reset everything from the admin section, I got error "Internal Server Error"
<img width="671" alt="image" src="https://user-images.githubusercontent.com/97154768/184195315-bd7f9b68-fb59-42db-82fd-62ad1fb5c606.png">

Server error: 

`PrismaClientKnownRequestError: Raw query failed. Code: `23503`. Message: `update or delete on table "event" violates foreign key constraint "event_data_event_id_fkey" on table "event_data"`
   ....
  code: 'P2010',
  clientVersion: '4.1.1',
  meta: {
    code: '23503',
    message: 'update or delete on table "event" violates foreign key constraint "event_data_event_id_fkey" on table "event_data"'
  }
}`

This happened due to the relation between event data and event  foreign key

**Comment by barissayil** [comment]
I have the same issue
```
db_1     | 2022-08-13 09:42:11.975 UTC [1150] ERROR:  update or delete on table "event" violates foreign key constraint "event_data_event_id_fkey" on table "event_data"
db_1     | 2022-08-13 09:42:11.975 UTC [1150] DETAIL:  Key (event_id)=(2) is still referenced from table "event_data".
db_1     | 2022-08-13 09:42:11.975 UTC [1150] STATEMENT:  delete from session where website_id=$1
umami_1  | PrismaClientKnownRequestError: Raw query failed. Code: `23503`. Message: `update or delete on table "event" violates foreign key constraint "event_data_event_id_fkey" on table "event_data"`
umami_1  |     at RequestHandler.handleRequestError (/app/node_modules/@prisma/client/runtime/index.js:28658:13)
umami_1  |     at RequestHandler.request (/app/node_modules/@prisma/client/runtime/index.js:28640:12)
umami_1  |     at async consumer (/app/node_modules/@prisma/client/runtime/index.js:29618:18)
umami_1  |     at async Proxy._request (/app/node_modules/@prisma/client/runtime/index.js:29639:16)
umami_1  |     at async __WEBPACK_DEFAULT_EXPORT__ (/app/.next/server/pages/api/website/[id]/reset.js:94:9)
umami_1  |     at async Object.apiResolver (/app/node_modules/next/dist/server/api-utils/node.js:184:9)
umami_1  |     at async NextNodeServer.runApi (/app/node_modules/next/dist/server/next-server.js:381:9)
umami_1  |     at async Object.fn (/app/node_modules/next/dist/server/base-server.js:491:37)
umami_1  |     at async Router.execute (/app/node_modules/next/dist/server/router.js:213:36)
umami_1  |     at async NextNodeServer.run (/app/node_modules/next/dist/server/base-server.js:610:29) {
umami_1  |   code: 'P2010',
umami_1  |   clientVersion: '4.1.1',
umami_1  |   meta: {
umami_1  |     code: '23503',
umami_1  |     message: 'update or delete on table "event" violates foreign key constraint "event_data_event_id_fkey" on table "event_data"'
umami_1  |   }
umami_1  | }
```

Similar problem with DELETE
```
db_1     | 2022-08-13 09:49:10.264 UTC [1165] ERROR:  update or delete on table "event" violates foreign key constraint "event_data_event_id_fkey" on table "event_data"
db_1     | 2022-08-13 09:49:10.264 UTC [1165] DETAIL:  Key (event_id)=(2) is still referenced from table "event_data".
db_1     | 2022-08-13 09:49:10.264 UTC [1165] STATEMENT:  DELETE FROM "public"."website" WHERE "public"."website"."website_id" IN ($1)
umami_1  | PrismaClientKnownRequestError: Foreign key constraint failed on the field: `event_data_event_id_fkey (index)`
umami_1  |     at RequestHandler.handleRequestError (/app/node_modules/@prisma/client/runtime/index.js:28658:13)
umami_1  |     at RequestHandler.request (/app/node_modules/@prisma/client/runtime/index.js:28640:12)
umami_1  |     at async consumer (/app/node_modules/@prisma/client/runtime/index.js:29618:18)
umami_1  |     at async PrismaClient._request (/app/node_modules/@prisma/client/runtime/index.js:29639:16)
umami_1  |     at async __WEBPACK_DEFAULT_EXPORT__ (/app/.next/server/pages/api/website/[id].js:146:9)
umami_1  |     at async Object.apiResolver (/app/node_modules/next/dist/server/api-utils/node.js:184:9)
umami_1  |     at async NextNodeServer.runApi (/app/node_modules/next/dist/server/next-server.js:381:9)
umami_1  |     at async Object.fn (/app/node_modules/next/dist/server/base-server.js:491:37)
umami_1  |     at async Router.execute (/app/node_modules/next/dist/server/router.js:213:36)
umami_1  |     at async NextNodeServer.run (/app/node_modules/next/dist/server/base-server.js:610:29) {
umami_1  |   code: 'P2003',
umami_1  |   clientVersion: '4.1.1',
umami_1  |   meta: { field_name: 'event_data_event_id_fkey (index)' }
umami_1  | }
```

**Comment by briancao** [comment]
Solved in https://github.com/umami-software/umami/pull/1449 and will be apart of the next release.

**Comment by mikecao** [comment]
Fixed in v1.38.0.
